xml_path: "/model/buff.xml"
bin_path: "/model/buff.bin"
rotate: false
twelve_classes: false
input_row: 416
input_col: 416
cof_threshold: 0.5
nms_area_threshold: 0.5
class_num: 12
